flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/DefaultDeclarativeSlotPoolTest.java

```diff
@@ -58,45 +61,66 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for the {@link DefaultDeclarativeSlotPool}. */
-class DefaultDeclarativeSlotPoolTest {
+@ExtendWith(ParameterizedTestExtension.class)
+class DefaultDeclarativeSlotPoolTest extends DefaultDeclarativeSlotPoolTestBase {
```

Comment:
Do we have test to check all requests are send in batch when interval > 0?

It means it's better to check the batch request take effect when interval > 0.